.github/workflows/ci.yml

```diff
@@ -0,0 +1,54 @@
+name: CI
+
+on:
+  push:
+    branches: [ "main", "master" ]
+  pull_request:
+    branches: [ "main", "master" ]
+
+permissions:
+  contents: read
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.10]
```

Comment:
Current latest stable Python is 3.14, and support for 3.14 was added to ansible-core 2.20 (supports Python 3.12 - 3.14 on the control node), so I suggest here 3.13, which is compatible with ansible-core 2.18-2.20